MetaMorpheus/GUI/TaskWindows/XLSearchTaskWindow.xaml.cs

```diff
@@ -306,6 +307,7 @@ private void SaveButton_Click(object sender, RoutedEventArgs e)
             }
 
             TheTask.XlSearchParameters.WritePepXml = ckbPepXML.IsChecked.Value;
```

Comment:
Do you want an option to update spectral library as well?

Reply:
If we end up needing that option, it will be a separate PR